ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -266,8 +267,12 @@ export const InteractiveNotebook = fp.flow(
         case RuntimeStatus.Deleted:
           return 'Preparing your Jupyter environment. This may take up to 5 minutes.';
         case RuntimeStatus.Error:
+          if (!enableCustomRuntimes) {
```

Comment:
isn't this logic flipped? The user can only see the compute configuration panel if enableCustomRuntimes is true.

Reply:
Oof, thanks for catching that. I swapped the logic around so it would be easier to remove the code, but seemingly failed to move the text.